cmd/wallet-web/src/pages/mixins/common/helper.js

```diff
@@ -89,6 +89,39 @@ export const isVPType = (type) => toLower(type) == 'verifiablepresentation';
 export const getCredentialType = (types) =>
   types.filter((type) => type != 'VerifiableCredential')[0];
 
+// function to get the credential display data
+export function getCredentialDisplayData(vc, manifest) {
+  // title
+  const title = populatePath(vc, manifest.title.path[0]) || manifest.title.fallback;
```

Comment:
why are you reading first path always??
should fallback to next one if first one doesn't match.

Reply:
There is only one path in each property

Reply:
No, there will be multiple, her sample has just one.